src/rars/tools/ConversionTool.java

```diff
@@ -5,6 +5,13 @@
 import javax.swing.event.DocumentListener;
 import java.awt.*;
 
+/**
+ * Add conversion for decimal, hexadecimal, binary and ascii tool to the toolbar
+ *
+ * @author Julián Hinojosa
+ * @version August 2024
+ **/
+
```

Comment:
This shouldn't be included.

Reply:
Ok, but the idea was to do a reference like in other files